src/main/java/com/Futurecollars/Lekcja4/Zadanie2/Square.java

```diff
@@ -0,0 +1,21 @@
+package com.Futurecollars.Lekcja4.Zadanie2;
+
+public class Square extends Shape {
```

Comment:
Niesformatowany kod, sformatuj sobie, żeby lepiej i czytelniej to wyglądało. Im więcej linii kodu będziesz miał w taki sposób zrobione tym mniej czytelny będzie kod.